### DISCUSSION TITLE
Transaction unspent - token not received

### DISCUSSION THREAD

**Discussion by pcsystemd** [dogecoin/dogecoin#3528] [Help needed]
Hello,

I hope this is the right place for this post if not a thousand apologies.

I don't know if it's a problem or not but as a test I sent a small quantity of Doge from a CEX to one of my addresses and I never received the tokens!

When I consult the explorer (see attached screenshots), I see that one of the transactions is "unspent" and I wonder if that's why I didn't receive anything!

I know it's a UTXO but I don't really know how it works since this is the first time I've seen it.

As far as you know, do I have to do anything in particular to get these tokens back?

Thank you in advance for your help.

![blockchair_info](https://github.com/dogecoin/dogecoin/assets/6862823/c95cb350-1923-4963-9433-98a9a343901b)
![dogechain](https://github.com/dogecoin/dogecoin/assets/6862823/4007dc3d-d30b-4be7-bb0e-9d2dc6c25487)
![dogechain_unspent](https://github.com/dogecoin/dogecoin/assets/6862823/0c739948-a467-4d34-9d12-6b35626ffb19)

**Comment by patricklodder** [comment]
> I sent a small quantity of Doge from a CEX to one of my addresses and I never received the tokens!

What wallet software do you use for this? Dogecoin Core?

**Reply by pcsystemd** [reply]
It's not Dogecoin Core I'm sure.

To be honest with you I can't remember who the receiving address belongs to as it's been over a month and I haven't as I usually do kept track but it was to another CEX,DEX or app I use. I've checked all the CEX,DEX, app, etc... that I use and I don't see any Doge token. I tried to find out which CEX, DEX, app, etc. it belonged to by looking at the reception address, but I couldn't find it!

So I'm afraid to send myself all my Doge tokens since I didn't receive the ones I sent for testing!

Thanks

**Reply by patricklodder** [reply]
If it's a CEX you'll need to check with them - the coin is sent, but not processed?

If it's a mobile wallet app, would help to know which app you were sending to.

I've independently confirmed on an index-everything node that the above transaction exists (it has > 35000 confirmations) and is still unspent (i.e. the coin didn't move)

**Reply by pcsystemd** [reply]
Thank you that's what I did but as a response I received this from CEX. Well I think these tokens are lost and in 5 years of cryptos this is the first time I've had this "unspent" problem. 
I contacted the CEX again, but without much hope!
—————-
Dear valued customer,
 
We would like to inform that your withdrawal of 619.6624173 DOGE has already been processed successfully on 2024-04-05 22:19 (UTC+8).
 
For any further assistance or inquiries related to your withdrawal, we recommend you to reaching out to your receiving platform and provide them the TXID so that they can help to locate your fund.
 
As blockchain transfers are impossible to cancel or recall when processed, we kindly suggest you contact your receiving platform for further assistance.
————

**Reply by patricklodder** [reply]
That's correct. The sending CEX has absolutely sent it as the transaction has been recorded on-chain. This is about the receiving wallet/CEX.
